lenskit-eval/src/main/java/org/lenskit/eval/traintest/predict/PredictEvalTask.java

```diff
@@ -298,7 +298,15 @@ public PredictConditionEvaluator(TableWriter tw, RatingPredictor pred, List<Metr
         public Map<String, Object> finish() {
             Map<String,Object> results = new HashMap<>();
             for (MetricContext<?> mc: predictMetricContexts) {
-                results.putAll(mc.getAggregateMeasurements().getValues());
+                MetricResult measurements = mc.getAggregateMeasurements();
+                // if test and train tests are disjoint we can get "null"
+                if (measurements != null) {
+                    results.putAll(measurements.getValues());
+                } else {
```

Comment:
The fix is exactly in these lines. You can revert my changes in ``PredictEvalTask`` and see how test is failing!